queries.sql

```diff
@@ -20,4 +20,12 @@ SELECT MAX(weight_kg) FROM animals WHERE species IN ('digimon');
 SELECT MIN(weight_kg) FROM animals WHERE species IN ('pokemon');
```

Comment:
### Optional
- [ ] It would be better if we indent the code well, it sure looks weird